src/instructlab/clickext.py

```diff
@@ -75,9 +75,8 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> click.Command | None
                 assert isinstance(cmd, click.Command)
             group, primary = self.get_alias_info(cmd_name)
             click.echo(
-                "You are using an aliased command, this will be deprecated "
-                "in a future release. Please consider using "
-                f"`ilab {group} {primary}` instead"
+                "You are using an aliased command. Please consider using "
+                f"`ilab {group} {primary}` instead."
```

Comment:
```suggestion
                "You used a deprecated command alias. Please use the replacement command "
                f"`ilab {group} {primary}` instead."
```

Reply:
We are keeping a few of the aliases, so I removed this error since they aren't part of the deprecated list.